app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,80 @@
+
+class TasksController < ApplicationController
+
+  def index
+    @tasks = Task.all.order(:id)
+  end
+
+  def show
+    task_id = params[:id].to_i
```

Comment:
Tiny note: actually, we don't need to do `.to_i` on this line anymore. Turns out, Rails will just accept `params[:id]` without the `.to_i` when we do `Task.find_by(id: ...)` because... Rails!